src/Illuminate/Contracts/Auth/MustVerifyEmail.php

```diff
@@ -31,4 +31,11 @@ public function sendEmailVerificationNotification();
      * @return string
      */
     public function getEmailForVerification();
+
+    /**
+     * Get the name of the "email verified at" column.
+     *
+     * @return string
+     */
+    public function getEmailVerifiedAtColumn(): string;
```

Comment:
We can't add methods to contracts on patch releases sorry. Please remove this and re-send the PR or send this to master, thanks.

Reply:
@driesvints I have to change target merge branch to master, right?